wordpress-mobile/WordPress-iOS
# Blaze: Hide dashboard card

Part of #20090

- Adds logic for hiding the Blaze dashboard card
- Analytics will be addressed in a different PR
- Figma: cEGtlsm6xijEswaKkrQrIj-fi-477%3A5675

## How to test
0. Make sure that Blaze is enabled via the debug menu
1. Switch to a site that has Blaze enabled - let's call this site A (See peeHDf-zm-p2 for more info on Blaze requirements)
2. Switch to the dashboard
3. ✅ A Blaze dashboard card is displayed
4. Tap on the ellipsis button and tap on "Hide this"
5. ✅ The Blaze dashboard card is hidden
6. Switch to another site hat has Blaze enabled - let's call this site B
7. ✅ A Blaze dashboard card is displayed
8. Switch back to site A
9. ✅ A Blaze dashboard card is NOT displayed


https://user-images.githubusercontent.com/6711616/220175750-51a4fdf2-85b1-4210-ac61-743dcef3dbe7.mp4



## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
